.circleci/config.yml

```diff
@@ -214,6 +214,7 @@ jobs:
             PRE_RELEASE=${CIRCLE_TAG/${CIRCLE_TAG%-rc[0-9]*}/}
             github-release delete -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -t ${CIRCLE_TAG} 2>/dev/null ||:
             ./hack/release-notes.sh ${CIRCLE_TAG} | github-release release ${PRE_RELEASE:+-p} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -t ${CIRCLE_TAG} -d -
+            sleep 3
```

Comment:
A more robust option would be to run `github-release info` in a loop until it succeeds. Eventual consistency can be quite unpredictable!

Reply:
thanks for the review. taking you suggestion i've added the command to check for the release with 5 retries in intervals of 1 second between retries.